apkid/rules/apk/protectors.yara

```diff
@@ -88,3 +88,19 @@ rule vkey_apk : protector
     1 of ($asseta*) and
     1 of ($assetb*)
 }
+
+rule free_rasp : protector
+{
+  meta:
+    description = "FreeRASP"
+    url         = "https://www.talsec.app/freerasp-in-app-protection-security-talsec"
+    sample      = "2b8faa038bf34474075a56e2fda7887a7df9c3c57db8a9f25547dc9374137ec9"
+    author      = "Fare9"
+
+  strings:
+    $lib1   = /lib\/(arm.*|x86.*)\/libsecurity\.so/
+    $lib2   = /lib\/(arm.*|x86.*)\/libpolarssl\.so/
```

Comment:
Don't you like the idea of having the `talsec` asset? If this file is not in all the APKs, we can always create two rules to differentiate compilers or versions.

Reply:
The application that @apkunpacker provided I think is an older version, after checking the diffing with the recent compiled versions I found that lacks some of the current protections, but I haven't found any versioning system, so probably the asset could be in newer versions, I wouldn't be 100% sure about it.

Reply:
In this case, it is recommended to keep both rules (one with the `talsec` asset file and the one with decryption routine based on DEX opcodes.